semgrep/semgrep/target_manager.py

```diff
@@ -106,11 +117,13 @@ def _parse_output(output: str, curr_dir: Path) -> Set[Path]:
                 )
             return files
 
-        def _find_files_with_extention(curr_dir: Path, extension: str) -> Set[Path]:
+        def _find_files_with_extention(
+            curr_dir: Path, extension: FileExtension
+        ) -> Set[Path]:
             """
                 Return set of all files in curr_dir with given extension
             """
-            return set(p for p in curr_dir.rglob(f"*.{ext}") if p.is_file())
```

Comment:
Was this broken before? `ext` wasn't a variable defined in this scope :O

Reply:
It wasn't broken :O the for loop this function was called in defined an ext